notNeededPackages.json

```diff
@@ -1866,6 +1866,12 @@
             "sourceRepoURL": "https://github.com/koajs/jwt",
             "asOfVersion": "3.3.0"
         },
+        {
+            "libraryName": "koa-useragent",
+            "typingsPackageName": "koa-useragent",
+            "sourceRepoURL": "https://github.com/rvboris/koa-useragent",
+            "asOfVersion": "2.1.1"
```

Comment:
Was this generated? The package readme puts the version at 2.1.0.

Reply:
This was generated based on this data 

https://gist.github.com/sandersn/d9ca982ff4d66be453d2c6f8f9be7fb9#file-not-needed-packages-txt-L714

```shell
Typings already defined for koa-useragent (koa-useragent) as of 2.1.1 (our version: 1.2)
  To fix this:
  git checkout -b not-needed-koa-useragent
  yarn not-needed koa-useragent 2.1.1 https://github.com/rvboris/koa-useragent
  git add --all && git commit -m "koa-useragent: Provides its own types" && git push -u origin not-needed-koa-useragent
  And comment PR: This will deprecate `@types/koa-useragent` in favor of just `koa-useragent`. CC @rvboris

```

cc @sandersn

Reply:
Gotcha. Turns out having the min version with the types is not important anyway. Thanks!